docs/create-candy/03-validate-assets.md

```diff
@@ -14,22 +14,22 @@ Ensure you have recent versions of `jq` installed:
 * [`jq` Installation](https://stedolan.github.io/jq/download/)
 
 ### 1. Check the number of assets (json and image files).
-Get count of total number of asset files:
+Get a count of the total number of asset files:
 ```bash
 $ find assets -type f  | wc -l
 ```
 
-Get count the metadata files
+Get a count of the metadata files
 ```bash
 $ find assets -type f -name '*.json' | wc -l
 ```
 
-Get count the images files
+Get a count of the image files
 ```bash
 $ find assets -type f -name '*.png' | wc -l
 ```
 
-For example, you have 10 images, and you will see next result after running the above commands:
+For example, if you have 10 images, you will see this result after running the above commands:
```

Comment:
In English, this construction would typically be expressed with an `if` clause like so.